<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GLWindow</class>
 <widget class="QMainWindow" name="GLWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GL View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="glWidget" native="true"/>
    </item>
    <item>
     <widget class="QSlider" name="xSlider">
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>15</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="ySlider">
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>15</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="zSlider">
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>15</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>15</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionFront">
   <property name="text">
    <string>Front (X+Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>Right (Y+Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="text">
    <string>Top (X+Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionAngled">
   <property name="text">
    <string>Angled</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionOrthogonal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Orthogonal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAuto_rotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto rotate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
